kitsune/products/tests/test_templates.py

```diff
@@ -93,10 +93,9 @@ def test_document_listing_order(self):
             doc.products.add(p)
             docs.append(doc)
 
-        # Add a helpful vote to the second document. It should be first now.
-        rev = docs[1].current_revision
-        helpful_vote(revision=rev, helpful=True, save=True)
-        docs[1].save()  # Votes don't trigger a reindex.
+        # Add a lower display order to the second document. It should be first now.
```

Comment:
Can you tweak the docstring of this test function to mention the display_order field too?